src/stuf/stuf_zds/templates/stuf_zds/soap/includes/bezoekadres.xml

```diff
@@ -0,0 +1,9 @@
+<BG:bezoekadres>
+    <BG:authentiek>N</BG:authentiek>
+    {% if initiator.verblijfsadres.woonplaatsNaam %}<BG:wpl.woonplaatsNaam>{{ verblijfsadres.woonplaatsNaam }}</BG:wpl.woonplaatsNaam>{% endif %}
```

Comment:
Wasn't sure if I should add an extra mapping `initiator.bezoekadres.woonplaatsNaam` instead of reusing the one for `verblijfsadres`